Datahub.Web/Pages/Asset.cshtml

```diff
@@ -130,7 +130,7 @@
                                     <div class="cell small-11">
                                         <div class="highlightable">
                                             <span>
-                                                <a href="@(d.Http.Url)"
+                                                <a href="@AssetHelpers.EnsureHttpsLink((d.Http.Url))"
```

Comment:
I'm not sure we can enforce this because some of the links will be to external sites (not to data.jncc.gov.uk) which may or may not have https

Reply:
Pushed new commit